### ISSUE TITLE
Where download older version?

### ISSUE THREAD

**Issue by valepe** [sqlitebrowser/sqlitebrowser#1822]
Where can I find versions older than 3.11?
I updated to the last one but regret to did it.
The import from csv now puts NULL values instead of empty text and I can't find a way to bypass this behavior. So I would like to downgrade to older version.
Portable app isn't a viable option because db files aren't associated to this app.

**Comment by justinclift** [comment]
Ouch, that doesn't sound good. :frowning: 

All of our prior releases are here:

https://github.com/sqlitebrowser/sqlitebrowser/releases

You might need to scroll down to the bottom to find the one you want.  I'm guessing this is the right one though:

&nbsp; &nbsp; https://github.com/sqlitebrowser/sqlitebrowser/releases/tag/v3.10.1

> The import from csv now puts NULL values instead of empty text and I can't find a way to bypass this behavior.

I **think** there's a way to do this in the Preferences, but it's been ages since I looked.  Gimme a minute... :smile:

**Comment by justinclift** [comment]
Hmmm, I'm looking at the commit which introduced a toggle option for it:

https://github.com/sqlitebrowser/sqlitebrowser/commit/17f1eabb65ee19346efac42c9533b3d6cff0903e

But when looking at the Import CSV dialog now, I'm not seeing the option.

Team: Anyone know if it's been renamed or something?

**Comment by MKleusberg** [comment]
Yeah, we experimented a lot with this. In commit e851c161d6c970a46e07cb0b222cf45d4ede3114 we removed the option @justinclift mentioned again.

With the latest version you can still get an empty string instead of a NULL value but you have to set the field to ```NOT NULL``` in your table schema. This is because we try to automatically detect a sensible default behaviour.

Example:
```sql
CREATE TABLE test1(
    a TEXT           -- DB4S will import empty values as NULL here
);
CREATE TABLE test2(
    a TEXT NOT NULL  -- DB4S will import empty values as empty string here
);
```

Does that work for you, @valepe? Or are you unable to change the table schema for your database?

**Comment by valepe** [comment]
I reinstalled the older version because I'm in hurry to finish some works.
I'll retry but same thing happened with a numeric (int) field. It was filled with null and not 0 (zero) as default value.
I'd like a checkbox in the import dialog to override the null behavior anyway
